xDripTests/Scenes/Entries/EntriesList/Logic/EntriesListTableControllerTests.swift

```diff
@@ -97,26 +85,24 @@ class EntriesListTableControllerTests: XCTestCase {
     func testDeleteEntry() {
         loadView()
         
-        sut.reload(with: [])
-        let data = generateDummyData()
-        sut.reload(with: data)
-        tableView.reloadData()
-        
-        let indexPath = IndexPath(row: 10, section: 0)
-        
-        sut.tableView(tableView, commit: .delete, forRowAt: indexPath)
-        
-        XCTAssertTrue(calledDelete)
-        XCTAssertTrue(tableView.numberOfRows(inSection: 0) == 19)
+        dispatchAfter(seconds: 1) {
```

Comment:
1. Isn't regular `DispatchQueue.main.async {}` enough? It should dispatch it on the next run loop, which should fix the issue.
2. If you add async operations, you also need to add expectations to prevent tests to end early.

Reply:
I've tried regular `DispatchQueue.main.async {}`, it didn't fix issue.

Reply:
Try a different approach, like adding it on a view controller first, similar to view controller unit tests.